libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -888,3 +887,11 @@ singleton AP_RTC depends AP_RTC_ENABLED
 singleton AP_RTC rename rtc
 singleton AP_RTC method clock_s_to_date_fields boolean uint32_t'skip_check uint16_t'Null uint8_t'Null uint8_t'Null uint8_t'Null uint8_t'Null uint8_t'Null uint8_t'Null
 singleton AP_RTC method date_fields_to_clock_s uint32_t uint16_t'skip_check int8_t'skip_check uint8_t'skip_check uint8_t'skip_check uint8_t'skip_check uint8_t'skip_check
+
+include AP_Networking/AP_Networking.h depends AP_NETWORKING_ENABLED
+singleton AP_Networking depends AP_NETWORKING_ENABLED
+singleton AP_Networking rename networking
```

Comment:
Needs another line to include the config
`include AP_Networking/AP_Networking_config.h`

Reply:
fixed